lib/sentiance/sentiance.go

```diff
@@ -54,3 +59,37 @@ type UserResponse struct {
 	ExternalID   string                          `json:"external_id"`
 	EventHistory []*EnrichTransportEventResponse `json:"event_history"`
 }
+
+type TransportEventWithWaypointsResponse struct {
+	Data DataWithWaypointsResponse `json:"data"`
+}
+
+type DataWithWaypointsResponse struct {
+	User UserWithWaypointsResponse `json:"user"`
+}
+
+type UserWithWaypointsResponse struct {
+	ID           string                                       `json:"id"`
+	ExternalID   string                                       `json:"external_id"`
+	EventHistory []*EnrichTransportWithWaypointsEventResponse `json:"event_history"`
+}
+
+type EnrichTransportWithWaypointsEventResponse struct {
+	EventID      string                             `json:"event_id"`
+	Type         string                             `json:"type"`
+	Mode         string                             `json:"mode"`
+	OccupantRole string                             `json:"occupant_role"`
+	Start        string                             `json:"start"`
+	End          string                             `json:"end"`
+	Waypoints    []EnrichTransportWaypointsResponse `json:"waypoints"`
+}
+
+type EnrichTransportWaypointsResponse struct {
+	Type      string    `json:"type"`
+	Latitude  int64     `json:"longitude"`
+	Longitude int64     `json:"latitude"`
+	Timestamp time.Time `json:"timestamp"`
+	Accuracy  int       `json:"accuracy"`
+	Speed     int64     `json:"speed"`
+	Altitude  int64     `json:"altitude"`
+}
```

Comment:
I think some of these struct names are confusing, but it isn't the end of the world either.

Reply:
Take any of the suggestions if you like 🤷